### PR TITLE
1.1.0 Release

### LINKED ISSUE #10: Include Captions in Lite Dataset
I noticed that in the Lite dataset, there is only an AI caption. Is there a reason that the user's submitted caption isn't there?

### LINKED ISSUE #13: Potential Data Cleanup activities
In the `unsplash_photos.photo_location_country` and `unsplash_photos.photo_location_city` the values appear to be freeform text that was probably direct user input, with effectively duplicate entries for example,

```
unsplash_lite=# select '>' || photo_location_city || '<' as city, '>' || photo_location_country || '<' as country,  count(*) from unsplash_photos where lower(photo_location_city) like '%london%' group by 1,2;
 ?column?  |       ?column?       | count
-----------+----------------------+-------
 >LONDON < | >United Kingdom <    |     1
 >London<  | >Canada<             |     7
 >London<  | >Egyesült Királyság< |     1
 >London<  | >England<            |     1
 >London<  | >U.K.<               |     2
 >London<  | >U.K<                |     1
 >London<  | >United Kingdom <    |     1
 >London<  | >United Kingdom<     |    73
 >London<  |                      |     3
(9 rows)```
```

It looks like there needs to be some data cleaning on these fields, definitely some stripping white space and such. Is it assumed that we should do our own location normalization on this and possibly add in a `normalized_photo_location_country` and `normalized_photo_location_city` ?

Also - over in `unsplash_conversion.conversion_country` this appears to be ISO 2 letter country codes. Is this guaranteed to be a valid ISO 2 letter country code? And was this data created based upon a maxmind geoip lookup or something similar?

Thanks so much for this dataset, I think it is going to be quite useful for demonstrational purposes. I hope these questions help increase the quality of what is a already great dataset.

### LINKED ISSUE #21: Add width, height and aspect ratio of the photos
The dataset is missing width, height and aspect ratio of the photo.

These are 3 important elements and should be appearing as 3 distinct fields.

### LINKED ISSUE #22: Image analysis metadata available?
Ticket #21 mentions additional image metadata that should be in the dataset. Are there other image analysis things that unsplash calculates that could be added? I'm in particular thinking of color value statistics,  mean/median pixel value, min/max pixel values etc.

### LINKED ISSUE #29: Clarification of embedded newlines in TSV files
The keywords data file appears to have an embedded newline in one of the records. I just want to clarify if this is expected or not.  It looks like the given psql loading instructions do account for newlines in the TSV file, but if folks are processing the file outside of that without using quote-escaping rules they may process the data incorrectly.

**To Reproduce**

```sh
% wc -l *.tsv000
1646598 collections.tsv000
4075505 conversions.tsv000
2689741 keywords.tsv000
25001 photos.tsv000
8436845 total
```
Load the according to the documented instructions:
```
 % psql -h localhost -U jeremy -d unsplash_lite -f load-data-client.sql
    COPY 25000
    COPY 2689739 # <-- Hmm.. this one is NOT 1 less than keywords.tsv000 above
    COPY 1646597
    COPY 4075504
```
Check the db row count
```
unsplash_lite=# select count(*) from unsplash_keywords;
  count
---------
 2689739
(1 row)
```

**Expected behavior**

I initially expected there to be 1 record for each non-header line of TSV, this appears to be an incorrect assumption. It looks like the psql commandline parsed the TSV according to quoted escape rules, so that is good.

 I wrote a [program to check the keywords file](https://gist.github.com/copiousfreetime/ab23addcb3a6e5612a77d0724e5d52b9) and it reports

```txt
% ruby check-tsv.rb keywords.tsv000
Headers: photo_id -- keyword -- ai_service_1_confidence -- ai_service_2_confidence -- suggested_by_user
[1590611 - PF4s20KB678-"fujisan] parts count 2 != 5
[1590612 - mount fuji"-] parts count 4 != 5
lines in file   : 2689741
data lines      : 2689740
unique row count: 2689740
```

Then looking at the lines around line 1590610 we see:

```txt
 % sed -n '1590610,1590615p' keywords.tsv000
PF4s20KB678     night   22.3271160125732                f
PF4s20KB678     "fujisan
mount fuji"                     t
PF4s20KB678     pier    22.6900939941406                f
PF4s20KB678     viaduct 30.6490669250488                f
PF4s20KB678     architecture    33.084938049316399              f
```

And the db reports that row and the preceding and following rows correctly loaded.

```txt
unsplash_lite=# select * from unsplash_keywords where photo_id = 'PF4s20KB678' and keyword like '%fujisan%';
  photo_id   |  keyword   | ai_service_1_confidence | ai_service_2_confidence | suggested_by_user
-------------+------------+-------------------------+-------------------------+-------------------
 PF4s20KB678 | fujisan   +|                         |                         | t
             | mount fuji |                         |                         |
(1 row)
unsplash_lite=# select * from unsplash_keywords where photo_id = 'PF4s20KB678' and keyword like '%pier%';
  photo_id   | keyword | ai_service_1_confidence | ai_service_2_confidence | suggested_by_user
-------------+---------+-------------------------+-------------------------+-------------------
 PF4s20KB678 | pier    |        22.6900939941406 |                         | f
(1 row)

unsplash_lite=# select * from unsplash_keywords where photo_id = 'PF4s20KB678' and keyword like '%night%';
  photo_id   | keyword | ai_service_1_confidence | ai_service_2_confidence | suggested_by_user
-------------+---------+-------------------------+-------------------------+-------------------
 PF4s20KB678 | night   |        22.3271160125732 |                         | f
```

If folks are processing these TSV simplistically without using quote-escaping logic then they may process the files incorrectly. I don't want folks to encounter that. And maybe this points to and upstream data input issue, if users are entering newlines in the keyword input - how are they getting processed in the main app.

We may just want to document that there can be embedded newlines in the TSV files.

Thanks!

### PR DESCRIPTION
**Pull request by TimmyCarbone**
#### Overview

- Closes #10 
- Closes #13  
- Closes #21 
- Closes #22
- Closes #29 

A 1.1.0 release tag will be created with more details about what's new in this version:

**New:**
- #10: Included user descriptions of the photos 
- #21: Included width, height and aspect ratio of photos
- #22: Included colors data from photos, coming from a 3rd party AI

**Fix:**

- #13: Trimmed [some fields](https://github.com/unsplash/datasets/issues/13#issuecomment-674709294).
- #29: Replaced newlines in keywords file by spaces to avoid CSV importation issues

### CODE REVIEW

**Comment by copiousfreetime** [comment]
@TimmyCarbone These look like some pretty fun updates!

File: DOCS.md [hunk 1]
```diff
@@ -78,6 +82,23 @@ The `conversions.tsv` dataset has one row per search conversion. The dataset tel
 | anonymous_user_id             | Anonymous user ID |
 | conversion_country            | Country code of the device geolocation |
 
+## 5 - colors.tsv
+
+*Note: The coverage and score data comes from a 3rd party AI*
+
+The `colors.tsv` dataset has one row per major color present in the photo. The dataset tells you which photo contains which colors and by how much its covering it as well as how much focus the color has (score).
```
Comment (lukechesser) [inline DOCS.md:89]: Maybe:

```suggestion
The `colors.tsv` dataset has one row per major color present in the photo. The dataset tells which colors are contained within a photo, their coverage as a percentage, and a score for how in focus the color is.
```

File: DOCS.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ The `photos.tsv` dataset has one row per photo. It contains properties of the ph
 | photo_image_url                | URL of the image file. Note: this is a [dynamic URL](https://unsplash.com/documentation#dynamically-resizable-images), so you can apply [resizing and customization operations directly on the image](https://unsplash.com/documentation#supported-parameters) |
 | photo_submitted_at             | Timestamp of when the photo was submitted to Unsplash |
 | photo_featured                 | Whether the photo was promoted to the [Editorial feed](https://unsplash.com/) or not |
+| photo_width                    | Width of the photo |
+| photo_height                   | Height of the photo |
```
Comment (lukechesser) [inline DOCS.md:17]:
```suggestion
| photo_width                    | Width of the photo in pixels |
| photo_height                   | Height of the photo in pixels |
```

**Review by lukechesser — APPROVED** [review]
Looking clean 👏

**Comment by lukechesser** [comment]
Also, mind copying over this PR description to the Changelog?

**Comment by TimmyCarbone** [comment]
Thanks @lukechesser!

Oh yeah, good catch on the changelog. It's kind of a duplicate of the release tags so I forgot about it.

**Pull request merged**